dataframely/columns/_base.py

```diff
@@ -85,6 +84,8 @@ def __init__(
         self.check = check
         self.alias = alias
         self.metadata = metadata
+        # The name may be overridden by the schema on column access.
+        self.name = alias or ""
```

Comment:
I think it would make sense for `name` to be only truly available if the column is accessed via the schema. Hence, I'd probably set this to `""` to begin with and disregard the alias. The fact that the alias is available here does not make sense conceptually IMO, it's simply the only place where one can specify the name of a column with a non-Python-identifier-name
```suggestion
        self.name = ""
```

Reply:
Also, would it make sense to make this `_name` and expose another property `name`? This would allow documenting it and making it obvious that it is similar to `col`

Reply:
c8570b5